src/src/pages/playlist/singlePlaylist/SinglePlaylist.jsx

```diff
@@ -0,0 +1,55 @@
+import "./SinglePlaylist.css";
+import React, { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import HistoryCard from "../../../pages/history/components/HistoryCard";
+import { useVideo } from "../../../context/provider/VideoProvider";
```

Comment:
you can use barrel export on these files

Reply:
Ok